### ISSUE TITLE
Item list have wrong font with "dark" themes

### ISSUE THREAD

**Issue by endorama** [sn-extensions/minimal-markdown-editor#5]
Hello, thank you for the product, is awesome and I really like using it!

I found a little inconsistency in how themes handle item list with Minimal Markdown editor extension.

Item lists are rendered with a non-monospace font. Here is a demonstration:
- correct
![selection_009](https://user-images.githubusercontent.com/526307/44594743-5b7c3f80-a7c7-11e8-82b6-3e518e096387.png)
- maybe wrong
![selection_008](https://user-images.githubusercontent.com/526307/44594757-6505a780-a7c7-11e8-8bb9-73657d9fb1c0.png)

Affected themes:
- Focus
- Futura
- Midnight
- Solarized Dark
- Titanium

**Comment by mobitar** [comment]
Yeah this is strange. Not sure what's causing this yet. Will need to investigate it further.
